warehouse/internal/repo/staging_test.go

```diff
@@ -231,6 +226,47 @@ func TestStagingFileRepo_Many(t *testing.T) {
 			})
 		}
 	})
+
+	t.Run("GetSchemasByIDs", func(t *testing.T) {
+		t.Parallel()
```

Comment:
[out of curiousity] Do we really need this `t.Parallel()` given that all the sub tests have one? :thinking:

Reply:
It's not required here. Removing it.